src/main/java/frc/robot/subsystems/Maflipulator.java

```diff
@@ -0,0 +1,143 @@
+package frc.robot.subsystems;
+
+import com.chopshop166.chopshoplib.outputs.SendableSpeedController;
+import edu.wpi.first.wpilibj.GenericHID.Hand;
+import edu.wpi.first.wpilibj.command.Command;
+import edu.wpi.first.wpilibj.command.Subsystem;
+import edu.wpi.first.wpilibj.interfaces.Potentiometer;
+import frc.robot.Robot;
+import frc.robot.RobotMap;
+
+public class Maflipulator extends Subsystem {
+
+    public enum MaflipulatorSide {
+        kFront, kBack;
+    }
+
+    private MaflipulatorSide currentPosition;
+
+    private SendableSpeedController flipMotor;
+    private Potentiometer manipulatorPot;
+
+    public Maflipulator(final RobotMap.MaflipulatorMap map) { // NOPMD
+        super();
+        flipMotor = map.getFlipMotor();
+        manipulatorPot = map.getManipulatorPot();
+    }
+
+    @Override
+    public void initDefaultCommand() {
+        // Set the default command for a subsystem here.
+        setDefaultCommand(manualFlip());
+    }
+
+    public Command manualFlip() {
+        // The command is named "Sample Command" and requires this subsystem.
+        return new Command("Manual Flip", this) {
+            @Override
+            protected void initialize() {
+                // Called just before this Command runs the first time
+            }
+
+            @Override
+            protected void execute() {
+                flipMotor.set(Robot.coPilot.getY(Hand.kRight));
+            }
+
+            @Override
+            protected boolean isFinished() {
+                return false;
+            }
+
+            @Override
```

Comment:
Delete this function and `interrupted`, since there's nothing special about them